### PR TITLE
feat(RELEASE-974): add push-disk-images-to-cdn pipeline

### PR DESCRIPTION
**Pull request by johnbieren**
@scoheb @davidmogar @ralphbean I do not know what tasks are supposed to be in this pipeline (do we still want advisories? do we still want to run push-snapshot to copy the images somewhere else in quay? do we still want to sign them?)

### CODE REVIEW

File: pipelines/push-disk-images-to-cdn/push-disk-images-to-cdn.yaml [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+          value: "$(tasks.collect-data.results.data)"
+        - name: snapshotPath
+          value: "$(tasks.collect-data.results.snapshotSpec)"
+        - name: pipelineRunUid
+          value: $(context.pipelineRun.uid)
+      workspaces:
+        - name: data
+          workspace: release-workspace
+      runAfter:
+        - check-data-keys
```
Comment (johnbieren) [inline pipelines/push-disk-images-to-cdn/push-disk-images-to-cdn.yaml:212]: This becomes `push-snapshot` if we are keeping that task in the pipeline

**Comment by mmalina** [comment]
@johnbieren you recently changed this to Ready. But what about the questions in the PR descriptions? Did you get your answers and you think this is what we want?

**Comment by johnbieren** [comment]
> @johnbieren you recently changed this to Ready. But what about the questions in the PR descriptions? Did you get your answers and you think this is what we want?

Based on my slack conversations with Ralph, we want to stick with the tasks I have in it for now and eventually add more later on. I still wouldn't mind @davidmogar agreeing with that on the PR in a comment, but :slightly_smiling_face:

**Review by mmalina — APPROVED** [review]

**Pull request merged**